document-generator-prosecution/src/main/java/uk/gov/companieshouse/document/generator/prosecution/mapping/mappers/ApiToDefendantMapper.java

```diff
@@ -10,15 +12,36 @@
 @RequestScope
 @Mapper(componentModel = "spring")
 public interface ApiToDefendantMapper {
+
+    @BeforeMapping
+    default void beforeMapping(@MappingTarget Defendant defendant, DefendantApi defendantApi) {
+        if(defendantApi.getIsCorporateAppointment()) {
```

Comment:
This doesn't need the get. Happy for this to be a change in a differnet story if the model is within the SDK.
```
defendantApi.isCorporateAppointment()
```

Reply:
This is in a model in the SDK, will raise a story for this to be changed.